uninstall.sh

```diff
@@ -16,4 +19,16 @@ fi
 
 # remove install.sh from rc.local
 STARTUP=$SCRIPT_DIR/install.sh
-sed -i "\~$STARTUP~d" $RC_LOCAL_FILE
\ No newline at end of file
+sed -i "\~$STARTUP~d" $RC_LOCAL_FILE
+
+# delete log folder in var log if they exist  
+if [ -d $LOG_DIR ]; then  
+    while true; do
```

Comment:
Wann hört die Schleife auf? Nur, wenn man "n" eingibt??

Reply:
Sicher? Das exit steht nur bei "Nn", das "break" gilt (glaube ich) nur für das case.

Reply:
Jetzt ist es richtig. Ich habe mich getäuscht. Richtig ist: Das "break" wirkt in bash nicht auf das case sondern (hier) auf die while loop.